### PR TITLE
5-get-reviews

### PR DESCRIPTION
**Pull request by tyr-ros**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -113,4 +113,68 @@ describe('GET: /api/reviews/:review_id', () => {
...
+            .get('/api/reviews')
+            .expect(200)
+    })
+    it('gives an array of objects with the following properties', () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res) => {
+                const body = res.body.reviews
+                expect(body).toBeInstanceOf(Array)
```
Comment (Jodiech) [inline __tests__/app.test.js:130]: Good testing here. You could be more thorough in checking the response data types by checking the length of the returned array of reviews and using a **forEach** to assert that all the properties are of the correct data type. Let me know if you need more info on this 👍

**Review by Jodiech — COMMENTED** [review]
Great work on this ticket 🚀 A few comments are dotted around, please see the comments below. I'd like you to submit another PR once you have made these changes. Give me a shout if you need any help!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -113,4 +113,68 @@ describe('GET: /api/reviews/:review_id', () => {
...
+    it("returns reviews sorted descendingly", () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res) => {
+                const body = res.body.reviews
+                expect(body[0].designer).toEqual('Avery Wunzboogerz')
+                expect(body[3].designer).toEqual('Asger Harding Granerud')
+                expect(body[7].designer).toEqual('Fiona Lohoar')
+            })
```
[the remark this answers is not in the document]
Comment (tyr-ros) [reply __tests__/app.test.js:167]: Completely forgot about the library, that is so much better to use!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -161,9 +161,9 @@ describe('GET: /api/reviews', () => {
             .expect(200)
             .then((res) => {
                 const body = res.body.reviews
-                expect(body[0].designer).toEqual('Avery Wunzboogerz')
-                expect(body[3].designer).toEqual('Asger Harding Granerud')
-                expect(body[7].designer).toEqual('Fiona Lohoar')
+                expect(body).toBeSortedBy('created_at', {
+                    descending: true,
+                });
```
Comment (leekli) [inline __tests__/app.test.js:168]: Excellent, thanks for implementing jest-sorted

File: controllers/reviews.controller.js [hunk 4]
```diff
@@ -11,10 +11,14 @@ exports.getReviewById = (req, res, next) => {
...
-    
     fetchReviews().then((reviews) => {
         res.status(200).send({
             'reviews': reviews
         })
+
+    })
+    .catch((err) => {
+        
+        next(err)
```
Comment (leekli) [inline controllers/reviews.controller.js:22]: Good adding the catch block to help with error handling

**Review by leekli — APPROVED** [review]
Hi Rebecca, well done and thanks for making these changes.

Good work and you are good to merge this branch.

**Pull request merged**